cwms-data-api/src/main/java/cwms/cda/api/watersupply/AccountingCatalogController.java

```diff
@@ -0,0 +1,202 @@
+import cwms.cda.data.dto.watersupply.WaterUser;
+import cwms.cda.data.dto.watersupply.WaterUserContract;
+import cwms.cda.formatters.ContentType;
+import cwms.cda.formatters.Formats;
+import cwms.cda.helpers.DateUtils;
+import io.javalin.core.util.Header;
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import java.time.Instant;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.http.HttpServletResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jooq.DSLContext;
+
+
+public class AccountingCatalogController implements Handler {
+    private final Logger LOGGER = Logger.getLogger(AccountingCatalogController.class.getName());
+    private static final String TAG = "Pump Accounting";
+    private static final String ROW_LIMIT = "row-limit";
+    private static final String ASCENDING = "ascending";
+    private final MetricRegistry metrics;
+
+    private Timer.Context markAndTime(String subject) {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+
+    public AccountingCatalogController(MetricRegistry metrics) {
+        this.metrics = metrics;
+    }
+
+    @NotNull
+    protected WaterSupplyAccountingDao getWaterSupplyAccountingDao(DSLContext dsl) {
+        return new WaterSupplyAccountingDao(dsl);
+    }
+
+    @OpenApi(
+        queryParams = {
+            @OpenApiParam(name = START, description = "The start time of the time window for "
+                + "pump accounting entries to retrieve. Defaults to the year 1800."),
+            @OpenApiParam(name = END, description = "The end time of the time window for pump "
+                + "accounting entries to retrieve. Defaults to the year 3000"),
+            @OpenApiParam(name = START_TIME_INCLUSIVE, description = "Whether or not the start time is "
+                + "inclusive or not. Defaults to TRUE.", type = Boolean.class),
+            @OpenApiParam(name = END_TIME_INCLUSIVE, description = "Whether or not the end time is inclusive "
+                + "or not. Defaults to TRUE.", type = Boolean.class),
+            @OpenApiParam(name = ASCENDING, description = "Whether or not the entries should be returned "
+                + "in ascending order. Defaults to TRUE.", type = Boolean.class),
+            @OpenApiParam(name = ROW_LIMIT, description = "The maximum number of rows to return. "
+                + "Defaults to 0, which means no limit.", type = Integer.class)
+        },
+        pathParams = {
+            @OpenApiParam(name = OFFICE, description = "The office ID the pump accounting is associated with.",
```

Comment:
This is the project's office right?

Reply:
Yes, updated the language to reflect that